core/templates/components/common-layout-directives/common-elements/promo-bar.directive.ts

```diff
@@ -62,10 +62,11 @@ angular.module('oppia').directive('promoBar', [
             setPromoDismissed(true);
           };
           ctrl.$onInit = function() {
-            PromoBarService.getPromoBarData().then(function(promoBarObject) {
-              ctrl.promoBarIsEnabled = promoBarObject.promoBarEnabled;
-              ctrl.promoBarMessage = promoBarObject.promoBarMessage;
-            });
+            PromoBarBackendApiService.getPromoBarDataAsync()
+              .then(function(promoBar) {
+                ctrl.promoBarIsEnabled = promoBar.promoBarEnabled;
```

Comment:
Please note that this PR has resulted in a serious breakage in production, since these two values are now returning `undefined`. This is because the promoBar object has promo_bar_enabled and promo_bar_message fields.

@biaba -- please make sure to test your PR manually on your local dev server before pushing changes. If you don't know how to test it, talk to your reviewer or mentor. But please **never** submit a PR that has not been tested manually, even though the changes might just look infrastructural.